Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "fedackings-lc-3-vm"
+version = "0.1.0"
+edition = "2024"
+
+[dependencies]
+termios = "0.3.3"
+timeout-readwrite = "0.4.0"
```

Comment:
In the version, locking the patch isn't needed nor desired. Those changes should never break the API of the dependency.

Reply:
just a comment for @SantiagoPittella, this is not locking the patch version, but setting a minimun patch version for the dependency